admin.js

```diff
@@ -0,0 +1,37 @@
+const config = require('./config/default.json');
+const levelup = require('levelup');
+
+const Commands = {
+  db: levelup(config.database, {valueEncoding: 'json'}),
+  db_get_all: function() {
+    this.db.createReadStream({
+      start: '\x00',
+      end: '\xFF'
+    })
+    .on('data', console.log)
+    .on('error', console.log);
+  },
```

Comment:
How about using a parameter like prefix or option?
Or we can make other function to search specific entity.